builtin/tag.c

```diff
@@ -103,9 +104,15 @@ static int delete_tag(const char *name, const char *ref,
 }
 
 static int verify_tag(const char *name, const char *ref,
-				const unsigned char *sha1)
+				const unsigned char *sha1, const char *fmt_pretty)
```

Comment:
hmm, I really don't see a problem with leaving it be. But wouldn't that just derive in another re-roll?